src/category_theory/limits/shapes/pullbacks.lean

```diff
@@ -148,16 +178,16 @@ abbreviation pullback_cone (f : X ⟶ Z) (g : Y ⟶ Z) := cone (cospan f g)
 namespace pullback_cone
 variables {f : X ⟶ Z} {g : Y ⟶ Z}
 
-def π₁ (t : pullback_cone f g) : t.X ⟶ X := t.π.app left
-def π₂ (t : pullback_cone f g) : t.X ⟶ Y := t.π.app right
+def fst (t : pullback_cone f g) : t.X ⟶ X := t.π.app left
```

Comment:
Should these be `def` or `abbreviation`?

Reply:
I'm uncertain. I haven't seen any problem so far with these being defs, but I haven't really exercised the pullbacks/pushouts stuff much. I say we leave these as is for now. At some point soon I will check that you can construct pullbacks from binary products and equalizers, and perhaps I'll see there if there's any issue.